docs/operations/licensing.md

```diff
@@ -78,55 +78,72 @@ Ondat cluster by browsing the __Licence__ page as shown below:
 
 ![license](/images/docs/concepts/licensing.png)
 
-## Ondat GUI - Obtaining a Licence via the GUI 
-
-Contact Ondat Customer Success team via getstarted@ondat.io and provide the
-following information:
-- **First** and **last** name
-- **Company** **name** and **role** (optional for Personal licence, mandatory
-  for Commercial licence)
-- **Cluster ID**
-
-In return to the registration, you will receive a licence file with a content
-similar to this one:
-
+## Ondat CLI - Running the CLI
+
+In order to get the CLI running, we must launch it on the target Kubernetes
```

Comment:
So, here it is not clear what is meant - who is "WE"? Is it an instruction to the dev? Also the fact the GUI is deleted and CLI added in its place confuses me. If that is an approved change by Rom it is fine by me, but the wording needs to be edited and changed. Everywhere we address the developer/client we write in the imperative form - "You need to do <instruction>"

Reply:
Thanks, great spot! Corrected.